lib/result.js

```diff
@@ -65,29 +64,24 @@ Result.prototype._parseRowAsArray = function (rowData) {
   return row
 }
 
-// rowData is an array of text or binary values
-// this turns the row into a JavaScript object
 Result.prototype.parseRow = function (rowData) {
```

Comment:
This is a simple `reduce` but I kept the same style as the array parser for consistency.